tests/foreman/api/test_location.py

```diff
@@ -61,13 +60,13 @@ class TestLocation:
     # TODO Add coverage for media, realms as soon as they're implemented
 
     @pytest.fixture
-    def make_proxies(self, options=None):
+    def make_proxies(self, target_sat, options=None):
```

Comment:
At the risk of being annoying, its so small that we should just fix it here. No fixtures should be taking kwargs with defaults - and nothing in this test module is trying to call this function.  This `options` kwarg needs to be removed.